### PR TITLE
[TS] [LPP] Root Folder configuration in the Document & Media widget is lost when using Category Filter.

### PR DESCRIPTION
**Pull request by thienquach87**
@tototrinh 

Here is the PR for fixing https://issues.liferay.com/browse/LPP-47633. Please help review and leave your comments.

Thanks,
Thien

### CODE REVIEW

File: modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/display/context/DLAdminDisplayContext.java [hunk 1]
```diff
@@ -913,6 +924,18 @@ _liferayPortletRequest, getSearchSearchContainerURL(), null,
 		return searchContainer;
 	}
 
+	private boolean _isAncestor(FileEntry fileEntry, long parentFolderId) {
```
Comment (tototrinh) [inline modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/display/context/DLAdminDisplayContext.java:927]: I think the function name should make more sense. May `_isAncestorFolder(long folderId, FilleEntry fileEntry)`? What do you think?

**Review by ces-totrinh — COMMENTED** [review]
Hi @thienquach87, I leave a comment. Please take a look and let me know what you think. Thanks.

File: modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/display/context/DLAdminDisplayContext.java (hunk 1 above)
Comment (thienquach87) [reply modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/display/context/DLAdminDisplayContext.java]: I updated it, thank you!

**Comment by ces-totrinh** [comment]
LGTM. Thanks.
Forwarded here: https://github.com/liferay-lima/liferay-portal/pull/3908

**Pull request closed without merging**